packages/material/src/layouts/MaterialArrayLayout.tsx

```diff
@@ -40,114 +56,266 @@ import Avatar from '@material-ui/core/Avatar';
 import Paper from '@material-ui/core/Paper';
 import DeleteIcon from '@material-ui/icons/Delete';
 import ValidationIcon from '../complex/ValidationIcon';
+import { connect, Dispatch } from 'react-redux';
+import { AnyAction } from 'redux';
 
-export const MaterialArrayLayout =
-  ({
-     data,
-     path,
-     label,
-     schema,
-     createDefaultValue,
-     uischema,
-     errors,
-     addItem,
-     findUISchema,
-     removeItems,
-     renderers
-   }: ArrayControlProps) => {
+const paperStyle = { padding: 10 };
+const iconStyle: any = { float: 'right' };
+export class MaterialArrayLayout extends React.Component<
+  ArrayLayoutProps,
+  { expanded: string | boolean }
+> {
```

Comment:
While this formatted correctly this is hard to read. I'd suggest to add some spacing, at least beween the class signature and its body.